x/profiles/client/cli/cli_app_links_test.go

```diff
@@ -0,0 +1,133 @@
+package cli_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/golang/protobuf/proto"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+
+	"github.com/desmos-labs/desmos/x/profiles/client/cli"
+	"github.com/desmos-labs/desmos/x/profiles/types"
+)
+
```

Comment:
The `GetCmdLinkApplication` tests are missing here 👀

Reply:
Yeah I know, unfortunately this is not an easy method to test. What we would have to do is: 
1. Instantiate a fake Band chain 
2. Create an IBC connection between the Desmos and Band chain 
3. Use that fake connection to send the packet using this command

Since this is very complicated, I think it's better to leave as it is without any test right now. I will take care of opening an issue and explaining what should be done once the PR gets merged

Reply:
Oh ok no problem!